integ/run-integ.ps1

```diff
@@ -75,8 +75,8 @@ $env:FLUENT_BIT_IMAGE = $FluentBitImage
 $env:TAG = -join ((65..90) + (97..122) | Get-Random -Count 10 | % {[char]$_})
 # AWS_FOR_FLUENT_BIT_CONTAINER_NAME is the name for the fluent-bit container ran for each service.
 $env:AWS_FOR_FLUENT_BIT_CONTAINER_NAME = "aws-for-fluent-bit-$($env:TAG)"
-$env:CW_INTEG_VALIDATOR_IMAGE = "906394416424.dkr.ecr.us-west-2.amazonaws.com/cw-integ-validator-windows"
-$env:S3_INTEG_VALIDATOR_IMAGE = "906394416424.dkr.ecr.us-west-2.amazonaws.com/s3-integ-validator-windows"
+$env:CW_INTEG_VALIDATOR_IMAGE = "906394416424.dkr.ecr.us-west-2.amazonaws.com/cw-integ-validator-windows:$Platform-$env:ARCHITECTURE"
+$env:S3_INTEG_VALIDATOR_IMAGE = "906394416424.dkr.ecr.us-west-2.amazonaws.com/s3-integ-validator-windows:$Platform-$env:ARCHITECTURE"
```

Comment:
@rawahars would there be a way to support a single Windows image for both 2019 and 2022 server versions?

Reply:
Yeah we can merge the digests of both images into a single manifest. Let's discuss this offline.

Reply:
From discussion, for windows images, it's better to build them each time. The base image updates every month for windows. The latest one will be cached on instance. The base image for windows is the largest part of the image. So re-build pulls in less data over network.